src/style.css

```diff
@@ -0,0 +1,11 @@
+.input_margin {
```

Comment:
Краще не вказувати в класі, що він робить
```suggestion
.input {
```

Reply:
запомню, спасибо!